docs/resources/dli_flinkjob_jar.md

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: "Data Lake Insight (DLI)"
+---
+
+# huaweicloud_dli_flinkjar_job
+
+Manages a flink job resource which type is `Flink Jar`  within HuaweiCloud DLI.
+
+## Example Usage
+
+### Create a flink job
+
+```hcl
+variable "name" {}
+variable "queue_name" {}
+variable "jar_obs_path" {}
+variable "entrypoint_args" {}
+
+resource "huaweicloud_dli_package" "test" {
+  group_name  = "jarPackage"
+  type        = "jar"
+  object_path = var.jar_obs_path
+}
+
+resource "huaweicloud_dli_flinkjar_job" "test" {
+  name            = var.name
+  queue_name      = var.queue_name
+  entrypoint      = "${huaweicloud_dli_package.test.group_name}/${huaweicloud_dli_package.test.object_name}"
+  entrypoint_args = var.entrypoint_args
+
+  runtime_config = {
+    foo = "bar"
+    key = "value"
+  }
```

Comment:
It is recommended to write the actual configuration, otherwise this confusing configuration should be deleted.